packages/provider/src/KnownRelaysManager.ts

```diff
@@ -11,22 +11,34 @@ import {
 import { LoggerInterface } from '@opengsn/common/dist/LoggerInterface'
 import { ContractInteractor } from '@opengsn/common/dist/ContractInteractor'
 import { isSameAddress } from '@opengsn/common'
-
-export const EmptyFilter: RelayFilter = (): boolean => {
+import { MAX_INTEGER } from 'ethereumjs-util'
+import { toBN } from 'web3-utils'
+import BN from 'bn.js'
+
+export const DefaultRelayFilter: RelayFilter = function (registeredEventInfo: RelayRegisteredEventInfo): boolean {
+  const maxPctRelayFee = 100
+  const maxBaseRelayFee = 1e17
```

Comment:
I am opposed to committing "magic numbers" in any case; 
At least make max- as constructor parameters, to allow easy override

Reply:
We MUST have a LOG to indicate that there was a relay and it was THROWN OUT, otherwise this is a silent censorship mechanism with a hard-coded configuration.

Reply:
I don't really care... we need to change it anyway into configurable parameter, but that's outside the scope of this issue, so its OK right know 
(the alternative is remove completely the new filtering mechanism until we implement the full configurable config - but I think its better to have this one.)